common/ObjectUtils.js

```diff
@@ -17,8 +17,9 @@ const createObjectFromDefaultProps = (overrideProps, defaultProps) => {
   if (!defaultProps) {
     throw new Error("Undefined `defaultProps`");
   }
+
```

Comment:
This change was made as previously returning default props avoided the `forEach` loop that correctly formatted the data as it's expected in the bots. For example, a liquidator bot with no `overrideProps ` provided would return the exact `defaultProps` object. This would be incorrectly formatted resulting in the bots throwing errors.